server/app/vectors.py

```diff
@@ -162,3 +163,40 @@ def add_leveln(
     raise HTTPException(
         status_code=status.HTTP_400_BAD_REQUEST, detail="Invalid vector"
     )
+
+
+@router.get("/genbank/{vector_id}", response_class=PlainTextResponse)
+def get_level1_genbank(
+    vector_id: int,
+    database: Session = Depends(deps.get_db),
+    # current_user: schemas.User = Depends(deps.get_current_user),
```

Comment:
```suggestion
    current_user: schemas.User = Depends(deps.get_current_user),
```

Reply:
current_user isn't used in this function.

Reply:
Yes. In general you can just prefix with an underscore. But this should raise some alarm bells. Without authentication, anyone can download any vector! Without checking if `current_user` is authorised then any logged in user can download any vector. Better check they're allowed access first.

Reply:
I agree that it's best to check for authorization but not with `current_user`.

E.g.: If user x (= `current_user`) makes a level 1 construct, using level 0 elements and a backbone of user y and z, he will not be able to generate a GenBank file of his level 1 construct.

Reply:
I don't really follow your reasoning. If `x` dies not have access to those backbones and level 0s how can they build the construct in the first place. Let alone download a genbank